packages/cli/src/gdal/gdal.docker.ts

```diff
@@ -51,7 +51,7 @@ export class GdalDocker extends GdalCommand {
   /** this could contain sensitive info like AWS access keys */
   private async getDockerArgs(): Promise<string[]> {
     const DOCKER_CONTAINER = Env.get(Env.Gdal.DockerContainer) ?? 'osgeo/gdal';
-    const DOCKER_CONTAINER_TAG = Env.get(Env.Gdal.DockerContainerTag) ?? 'ubuntu-small-3.3.0';
+    const DOCKER_CONTAINER_TAG = Env.get(Env.Gdal.DockerContainerTag) ?? 'ubuntu-small-3.6.1';
```

Comment:
This needs to be updated in the docketfile too

Reply:
Just updated. Thank you.